docker/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM golang:alpine
+RUN apk update && \
+    apk upgrade && \
+    apk add git python3 nodejs-npm bash && \
+    npm install -g less less-plugin-clean-css && \
+    git clone https://github.com/transceptor-technology/siridb-http.git /tmp/siridb-http && \
```

Comment:
@joente please hold merging: this will always build the latest master, which is not what we should do. I'll fix it asap